.github/workflows/check-opte-ver.yml

```diff
@@ -10,6 +10,8 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
+        with:
+          ref: ${{ github.event.pull_request.head.sha }} # see omicron#4461
```

Comment:
Just a general comment: for something that's not a workaround, I like to put the explanation directly in a comment rather than reference an issue.  Up to you -- it's not important to change this here.

Reply:
Ah I generally do agree, but in this case it was just that it needed to be referenced in a bunch of places. I'll leave it as is.